packages/alpine.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Alpine < Package                 # The first character of the class name must be upper case
+  description 'An easy-to-use email client.'
+  homepage 'http://alpine.x10host.com/alpine/'
```

Comment:
If this is available via https, please update to use the secure version.

Reply:
No HTTPS available